supabase/migrations/20230509035114_create_access_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE access (
+    user_name character varying(255) NOT NULL PRIMARY KEY,
+    repository text,
+    priority_access  boolean,
+    time_access boolean,
+    price_access boolean,
+    multiplier_access boolean,
+    created_at text,
+    updated_at text
```

Comment:
Lets make the type of both `timestamp`. Please take a look for the reference.  https://github.com/ubiquity/bounty-bot/blob/development/supabase/migrations/20230207013753_update_types.sql